InventoryInStorePickupMultishipping/Test/Unit/Controller/Checkout/PluginTest.php

```diff
@@ -40,7 +40,7 @@ protected function setUp(): void
     {
         $this->cartMock = $this->createMock(Cart::class);
         $this->quoteMock = $this->getMockBuilder(Quote::class)
-            ->addMethods(['setIsMultiShipping', 'getIsMultiShipping'])
+            ->setMethods(['setIsMultiShipping', 'getIsMultiShipping'])
```

Comment:
`setMethods` is deprecated. You should use either `addMethods` or `onlyMethods`, but as long as it's not throwing warning, you don't need to care about this too much (however it's being marked as deprecated with annotation already)

Reply:
I see, just found that test is failing due to method absence

Reply:
So we need to replace `onlyMethods` too :-( I don't know why you want to keep the compatibility with PHPUnit 6, as this is PR to PHP 7.4 branch, that drops backward compatibility with 6.

Reply:
I agree with you. Reverted the changes